docs/sdk/api-reference.md

```diff
@@ -3317,18 +2595,10 @@ Initializes a Search object.
 - `token` (*str*) - Session token
 - `root_certificates_path` (*str*) - Path to SSL certificates
 
-**Raises:**
-- UserError if:
-  - Invalid metric specified
-  - Invalid algorithm specified
-  - Cosine metric used with nearest neighbor
-  - Both top_k and pagination enabled
-
-
-### Search._get_annot_proto()
+### Search._get_annot_proto
```

Comment:
I think this is a internal method. WDYT @sanjaychelliah ?

Reply:
Yes

Reply:
As this is an internal method i have removed this from the doc